tpl-implementations/kokkos-kernels/include/blas1_scale_kk.hpp

```diff
@@ -0,0 +1,23 @@
+
+#include<experimental/mdspan>
+#include<experimental/linalg>
+#include<KokkosBlas.hpp>
+#include<exec_policy_wrapper_kk.hpp>
+namespace KokkosKernelsSTD {
+template<class ExecSpace,
+         class Scalar,
+         class ElementType,
+         std::experimental::extents<>::size_type ... ext,
+         class Layout,
+         class Accessor>
+void scale(kokkos_exec<ExecSpace>, const Scalar alpha,
+           std::experimental::mdspan<ElementType, std::experimental::extents<ext ...>, Layout, Accessor> x)
+{
+//  if constexpr(x.rank()==2) {
+    Kokkos::View<ElementType*> x_v(x.data(),x.extent(0));
```

Comment:
This is only correct if the layout is contiguous.

Reply:
yeah I know. I need to write a full proper mapper from mdspan to view. However I might not do that for now here, and wait until Kokkos::View is based on mdsapn in which case I can simply construct it directly. Until then I will just support the basic case of contiguous LayoutLeft and LayoutRight.

Reply:
We discussed this over voice this morning -- I'm fine with just deferring that task until `Kokkos::View` uses `mdspan`.  Thanks! : - )